eventbusk/bus.py

```diff
@@ -96,6 +96,7 @@ def __init__(self, broker: str):
         self._topic_to_event: dict[str, str] = {}
         self._event_to_topic: dict[str, str] = {}
         self._receivers: set[ReceiverWrappedT] = set()
+        self.post_receive_hook: Callable[[Event, bool], None] | None = None
```

Comment:
@GauthamGoli 

What kind of things do you foresee the post_receive_hook doing?  Can you give me some examples? 

Not sure if this is something the framework should handle because this could get hairy and generate more issues to deal with if people start adding complex post-receive methods.
Since a consumer can receive/process millions of events while its up, we can't shouldn't do much cleanup per event like db connection handling. The post hook event needs to be ultra-fast, and allowed to fail as i see in the handling below. Then is it of any use? Is it plainly to add logging? Can't it be added as the last line of the consumer code itself? What does a separate handler give us?